src/Spectre.libGenetic/GeneticAlgorithmFactory.h

```diff
@@ -43,17 +43,16 @@ class GeneticAlgorithmFactory
                             double bitSwapRate,
                             double preservationRate,
                             unsigned generationsNumber,
-                            unsigned numberOfCores,
-                            size_t minimalFillup,
-                            size_t maximalFillup);
+                            unsigned numberOfCores);
```

Comment:
Please remove the documentation on `minimalFillup` and `maximalFillup`

Reply:
done